.github/workflows/mergify-test.yml

```diff
@@ -1,12 +1,36 @@
+name: Proto merge integration
```

Comment:
Can you please rename `mergify-test.yml` to something better?  It is just debris left behind from a prior test that should be cleaned up.

Reply:
I was thinking of dropping the `mergify-test.yml` file completely once I've confirmed it does what I intend on regular PRs and master, and upgrade at least `deployment-test` to use this new logic (but possibly `integration-test` as well)